rpc/src/responses/block_ids_in_chain.rs

```diff
@@ -47,19 +23,37 @@ mod test {
     #[test]
     fn get_blocks_in_chain_from_response_empty_ok() {
         let mock_response = "";
-        let blocks = BlocksInChainResponse::from_response_str(mock_response);
-        assert!(blocks.block_ids.len() == 0);
+        let blocks_response = BlocksInChainResponse::from_response_str(mock_response);
+        let blocks = blocks_response.block_ids.into_vec();
+        assert!(blocks.is_empty());
+    }
+
+    #[test]
+    fn get_blocks_in_chain_from_empty_list_ok() {
+        let mock_response = "[]";
+        let blocks_response = BlocksInChainResponse::from_response_str(mock_response);
+        let blocks = blocks_response.block_ids.into_vec();
+        assert!(blocks.is_empty());
+    }
+
+    #[test]
+    #[should_panic]
+    fn get_blocks_in_chain_from_malformed_response_fails() {
+        let mock_response = "[[]]";
+        BlocksInChainResponse::from_response_str(mock_response);
     }
 
```

Comment:
a test where mock_response is `["blockId1"]` should be useful

Reply:
Good one, added. 

Since we're basing the tests off of the exact RPC spec, i'm marking it as a `#[should_panic]` since the format of `["..."]` is _not_ up to spec.